app/routes/docs.client.samples.md

```diff
@@ -164,3 +164,61 @@ for message in consumer.consume(end[0].offset - start[0].offset, timeout=1):
             continue
     print(message.value())
 ```
+
+## Working With JSON Schema
+
+For new missions, GCN Notices are preferably distributed in JSON format. This guide describes how to programmatically read the JSON schema.
+
+## Parsing JSON
+
+Read the JSON data from [sample.schema.json](https://gcn.nasa.gov/docs/notices/schema) and [sample.example.json](https://gcn.nasa.gov/docs/notices/schema), which parses it into Python dictionaries.
+
+```python
+from gcn_kafka import Consumer
+import json
+
+# Connect as a consumer
+consumer = Consumer(client_id='fill me in',
+                    client_secret='fill me in')
+
+# Subscribe to topics and receive alerts
+consumer.subscribe(['gcn.classic.voevent.FERMI_GBM_SUBTHRESH'])
+
+# Continuously consume and parse messages as JSON
+for message in consumer.consume(timeout=1):
+    if not message.error():
+        json_data = json.loads(message.value().decode('utf-8'))
+        print("Received JSON Notice:", json_data)
+```
+
+## Encoding and Decoding of Embedded Data
+
+The following code sample shows how to encode/decode a file in Python. The [`base64`](https://docs.python.org/3/library/base64.html#base64.b64encode) package includes the methods `b64decode` and `b64encode` to make this task simple.
+
+```python
+import base64
+
+# Parse the content of your file to a base64 encoded string:
+with open("path/to/your/file", 'rb') as file:
+    encoded_string = base64.b64encode(file.read())
+
+print(encoded_string)
+# b'a1512dabc1b6adb3cd1b6dcb6d4c6......'
+
+# Decode and write the content to a local file:
+with open("path/to/destination/file", 'wb') as file:
+    file.write(base64.b64decode(encoded_string))
+
+```
+
+If you want to include a FITS file in a Notice, you add a property to your schema definition in the following format:
+
+```python
+{
+    type: 'string',
+    contentEncoding: 'base64',
+    contentMediaType: 'image/fits',
+}
+```
+
+In your data production pipeline, you can use the encoding steps to convert your file to a bytestring and set the value of the property to this bytestring. See [non-JSON data](https://json-schema.org/understanding-json-schema/reference/non_json_data.html) for more information.
```

Comment:
Great sample code, but not enough context here to understand what these instructions are for.

Reply:
I have tried to expand the instructions. 
Could you please elaborate more what's missing?

Reply:
Pick an example of an actual notice type that contains embedded HEALPix data. Show how to:

- subscribe to and receive that notice type
- decode the Kafka message as JSON
- decode base64 text to bytes
- read the bytes as a FITS file

And then the user is ready to go on to the sky maps section! (I think that this is turning into a step-by-step tutorial, which will be awesome.)